scipy/special/tests/test_mpmath.py

```diff
@@ -1211,17 +1221,15 @@ def test_hermite(self):
 
     # hurwitz: same as zeta
 
-    @nonfunctional_tooslow
     def test_hyp0f1(self):
         assert_mpmath_equal(sc.hyp0f1,
                             _exception_to_nan(lambda a, x: mpmath.hyp0f1(a, x, **HYPERKW)),
-                            [Arg(), Arg()])
+                            [Arg(-10, 1e7), Arg(0, 1e5)])
```

Comment:
-10 is now -1e7 via the reflection relation for Bessel functions. The second argument here still starts from zero because z<0 part needs separate work.